core/src/mindustry/world/blocks/power/Battery.java

```diff
@@ -1,10 +1,41 @@
 package mindustry.world.blocks.power;
 
+import arc.*;
+import arc.graphics.Color;
+import arc.graphics.g2d.*;
+import mindustry.world.*;
+
 public class Battery extends PowerDistributor{
 
+    public TextureRegion baseRegion;
```

Comment:
The base region is just a white square, so it can be drawn with `Fill.square()`.

Reply:
ok